From a01489b677b8a3dde0245ec6753a401660ae1e29 Mon Sep 17 00:00:00 2001
From: Antony Vance <antony.vance@ni.com>
Date: Sun, 21 Feb 2016 01:18:05 +0530
Subject: [PATCH] opencv : Download ippicv from NI server

CMake is not able to download ippicv package from sourceforge https server due to disabled openssl option. This breaks opencv build with download hash mismatch error.
This change downloads ippicv pacakge from NI http server  to avoid build error.

Signed-off-by: Antony Vance <antony.vance@ni.com>
---
 3rdparty/ippicv/downloader.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/3rdparty/ippicv/downloader.cmake b/3rdparty/ippicv/downloader.cmake
index a6016db..ab1f12e 100644
--- a/3rdparty/ippicv/downloader.cmake
+++ b/3rdparty/ippicv/downloader.cmake
@@ -64,7 +64,7 @@ function(_icv_downloader)
       if(DEFINED ENV{OPENCV_ICV_URL})
         set(OPENCV_ICV_URL $ENV{OPENCV_ICV_URL})
       else()
-        set(OPENCV_ICV_URL "https://raw.githubusercontent.com/Itseez/opencv_3rdparty/${IPPICV_BINARIES_COMMIT}/ippicv")
+        set(OPENCV_ICV_URL "http://download.ni.com/ni-linux-rt/src/")
       endif()
     endif()
 
-- 
2.9.3

